raiden/tests/integration/rpc/test_assumptions.py

```diff
@@ -120,8 +107,7 @@ def test_transact_throws_opcode(deploy_client, blockchain_backend):
     address = contract_proxy.address
     assert deploy_client.eth_getCode(address) != '0x'
 
-    gas = contract_proxy.fail.estimate_gas()
-
+    gas = min(contract_proxy.fail.estimate_gas(), deploy_client.gaslimit())
```

Comment:
please update the docstring/name of the these tests:

- `test_transact_opcode`
- `test_transact_throws_opcode`
- `test_transact_opcode_oog`

the function `check_transaction_threw` is not relying on the opcodes anymore

Reply:
Do you know why `estimate_gas` returned a value larger than the block's gas limit? I wonder if this could happen with a valid transaction/call with production code.